src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -872,5 +882,31 @@ private static NodeView GetNotConnectedNode(Window stepMainWindow, string nodeTy
             return byOriginUserAdded;
         }
 
+
+        internal static void SubscribeRunButtonNextStep(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            if (uiAutomationData.Parameters.Any() && uiAutomationData.Parameters[0] is string)
+            {
+                Button runButton = GuideUtilities.FindChildInVisualTree(stepInfo.MainWindow, uiAutomationData.Parameters[0].ToString()) as Button;
+
+                if(runButton != null)
```

Comment:
For not having several nested if statements you can use something like:
if(runButton == null) return;